app/cpt_file/parametrization.py

```diff
@@ -14,55 +14,179 @@
+    OptionListElement,
+    Parametrization,
+    SetParamsButton,
+    Text,
+    TableInput,
+    And,
+)
+from .constants import (
+    DEFAULT_MIN_LAYER_THICKNESS,
+    DEFAULT_CLASSIFICATION_TABLE,
+    DEFAULT_SOIL_NAMES,
+    MAX_CONE_RESISTANCE_TYPE,
+)
 from .constants import DEFAULT_ROBERTSON_TABLE
 
 
-def _get_soils_options(**kwargs) -> List[OptionListElement]:
-    """Options all possible soil type from the Classification parameter in the Project entity."""
-    return [OptionListElement(soil.get('ui_name')) for soil in DEFAULT_ROBERTSON_TABLE]
+CLASSIFICATION_METHODS = [
+    OptionListElement(label="Robertson Method (Fugro)", value="robertson"),
+    OptionListElement(label="Table Method", value="table"),
+]
 
 
 class CPTFileParametrization(Parametrization):
     """Defines the input fields in left-side of the web UI in the CPT_file entity (Editor)."""
-    gef = Tab('GEF')
-    gef.text = Text("Use the table below to change the interpreted soil layout by changing the positions of the layers, "
-                    "adding rows or changing the material type.")
-
-    gef.filter_thin_layers = SetParamsButton('Filter Layer Thickness',
-                                             method='filter_soil_layout_on_min_layer_thickness', flex=60,
-                                             description="Filter the soil layout to remove layers that are "
-                                                         "thinner than the minimum layer thickness")
-    gef.min_layer_thickness = NumberField('Minimum Layer Thickness', suffix='mm', min=0, step=50,
-                                          default=DEFAULT_MIN_LAYER_THICKNESS,
-                                          flex=40)
-
-    gef.reset_original_layers = SetParamsButton('Reset to original Soil Layout',
-                                                method='reset_soil_layout_user',
-                                                flex=100,
-                                                description="Reset the table to the original soil layout" )
-
-    gef.ground_water_level = NumberField('Phreatic level', name='ground_water_level', suffix='m NAP', flex=50)
-    gef.ground_level = NumberField('Ground level', name='ground_level', suffix='m NAP', flex=50)
-    gef.soil_layout = TableInput('Soil layout', name='soil_layout')
-    gef.soil_layout.name = OptionField("Material", options=_get_soils_options)
-    gef.soil_layout.top_of_layer = NumberField("Top (m NAP)", num_decimals=1)
+
+    classification = Step("Upload and classification")
+    classification.text_01 = Text(
+        """# Welcome to the CPT interpretation app!
+
+With this app you will be able to classify and interpret GEF-formatted CPT files by uploading and automatically 
+classifying the soil profile based on the classification table.
+
+## Step 1: Upload a GEF file
+
+For the users who want to try out the app, but do not have a GEF file at hand, feel free to use the sample 
+GEF file available.
```

Comment:
The tab has been replaced with steps. Only one tab was used. Is that what you mean?

Reply:
oh i mean "spaces"? This text is fully lined out with the left margin. I don't really mind, it can be nicer to have the space, I'm just not used to it ;p

Reply:
Oh I understand what you mean. The tab can produce in some instances weird formatting, so to avoid this, I have done this consistently on all text.